sensu/plugins/check-vyatta-tunnels.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python
+#
+# Checks that all tunnels are up
+#
+# Return CRITICAL if any tunnel is down
+#
+# Dean Daskalantonakis <ddaskal@us.ibm.com>
+# Carl Hannusch <channusch@us.ibm.com>
+
+import argparse
+import re
+import subprocess
+import sys
+
+STATE_OK = 0
+STATE_WARNING = 1
+STATE_CRITICAL = 2
+CRITICALITY = 'critical'
+
+
+def switch_on_criticality():
+    if CRITICALITY == 'warning':
+        sys.exit(STATE_WARNING)
+    else:
+        sys.exit(STATE_CRITICAL)
+
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('-ip', '--ip', help='vyatta vrrp ip',
+                        required=True)
+    parser.add_argument('-p', '--password', help='snmp password',
+                        required=True)
+    parser.add_argument('-z', '--criticality', help='check criticality',
+                        required=True)
```

Comment:
I think we want to not make this required, that it should have a default of `'critical'`, like in `check-neutron-agents-duplicate.py`.